packages/app/src/specs/InlineSpecListTree.vue

```diff
@@ -156,6 +170,28 @@ const resetFocusIfNecessary = (row, index) => {
   }
 }
 
+const directoryChildren = computed(() => {
+  return collapsible.value.tree.reduce<Record<string, string[]>>((acc, node) => {
```

Comment:
Changed to use built-in `Record` type: https://github.com/cypress-io/cypress/pull/24738/commits/6618f603a1f184b2f6663d88c0282a9416114498